src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -195,8 +220,7 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     width: '100%',
-    flex: 1,
-    marginEnd: 4
+    flex: 1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The marginEnd property has been removed from the style. If this was intentional, ensure that this does not negatively impact the layout of the component in any scenario.



<!-- This is an auto-generated comment by OSS CodeRabbit -->